packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -47,10 +51,10 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
     /// @notice A mapping of withdrawal hashes to `ProvenWithdrawal` data.
     mapping(bytes32 => ProvenWithdrawal) public provenWithdrawals;
 
-    /// @notice Determines if cross domain messaging is paused.
-    ///         When set to true, withdrawals are paused.
-    ///         This may be removed in the future.
-    bool public paused;
+    /// @custom:legacy
+    /// @custom:spacer paused
+    /// @notice Spacer for backwards compatibility.
+    bool private spacer_53_0_1;
```

Comment:
Just curious why the name `spacer_53_0_1`, what does the `53_0_1` part refer to?

Reply:
slot: 53
offset: zero
bytes: 1

It should match the entries in the storage layout:

https://github.com/ethereum-optimism/optimism/blob/feat/pause-all/packages/contracts-bedrock/.storage-layout#L71